src/services/EthService/nonce.ts

```diff
@@ -3,5 +3,5 @@ import { ProviderHandler } from './network';
 
 export function getNonce(network: Network, address: TAddress) {
   const provider = new ProviderHandler(network);
-  return provider.getTransactionCount(address);
+  return provider.getTransactionCount(address) || 0;
```

Comment:
Is this safe?

Reply:
Hmm, I added it since:
1. SendAssets unit test need mock `ethers`
2. If I mock `ethers` getNonce result is `undefined` and everything crumbles.
3. Setting a default nonce value seemed reasonable. Unsure if this is the right way to do so.

Reply:
I'm unsure of whethyer a default nonce is a good idea or not. Can we not mock the `getTransactionCount`?

Reply:
From slack conversation:
> Emmanuel: I didn't find a way to mock provider:
couldn't mock a relative import inside dependency from within the test file.
couldn't find how to create an implementation of provider from within an ethers mock implementation https://github.com/MyCryptoHQ/MyCrypto/blob/285f33ed5be4c61c3c42fef91d60277853e62411/src/services/EthService/network/providerHandler.ts#L24

> Frederik: Not entirely sure, just feels weird to return something that's probably not true
Like in case of an error your nonce would be set to 0 and the tx would fail
without notifying the user that something is wrong

@blurpesec Any idea how to mock Provider?